### PR TITLE
Make it easier to run local version of PyPI.

### PR DESCRIPTION
**Pull request by Carreau**
Handle gracefully when one can't connect to s3.

Allow custom port and to deactivate TLS

---

This is just a cleanup of the local patch @michaelpacer  and I are using on top of #506 to test/run locally.

### CODE REVIEW

File: store.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 import sys, os, re, time, hashlib, random, types, math, stat, errno
 import logging, string, datetime, calendar, binascii, urllib, urllib2, cgi
 import posixpath
 from collections import defaultdict
 import cPickle as pickle
-try:
-    import psycopg2
-except ImportError:
-    pass
+import psycopg2
```
Comment (Carreau) [inline store.py]: I removed the except-pass anyway if fails later if one does not have psycopg2 working.

File: webui.py [hunk 2]
```diff
@@ -385,17 +386,20 @@ def __init__(self, handler, env):
...
-        self.package_bucket = self.s3conn.get_bucket(
-            self.config.database_files_bucket,
-            validate=False,
-        )
+            self.package_bucket = self.s3conn.get_bucket(
+                self.config.database_files_bucket,
+                validate=False,
+            )
+        else:
+            print("AWS database not fully configured. Getting the package files will be unavailable.", file=sys.stderr)
```
Comment (Carreau) [inline webui.py]: Note that this is mostly indenting, so viewing the diff with [`?w=1` ](https://github.com/pypa/pypi-legacy/pull/536/files?w=1) might make things easier to review.

File: README.rst [hunk 3]
```diff
@@ -33,6 +33,19 @@ Then you can launch the server using the pypi.wsgi script::
 
 PyPI will be available in your browser at http://localhost:8000
 
+You can use the `--port` option to change the port, in particular some browser
```
Comment (takluyver) [inline README.rst]: browsers

**Comment by Carreau** [comment]
Rebased on master. And fixed spelling.

**Pull request closed without merging**